esp/bindings/http/platform/httptransport.cpp

```diff
@@ -774,7 +774,14 @@ void CHttpMessage::logMessage(MessageLogFlag messageLogFlag, const char *prefix)
             else if ((m_content_type.length() > 0) && (strieq(m_content_type.get(), "text/css") || strieq(m_content_type.get(), "text/javascript")))
                 DBGLOG("%s<content_type: %s>", prefix, m_content_type.get());
             else
-                logMessage(m_content.str(), prefix);
+            {
+                StringBuffer httpPath;
+                getPath(httpPath);
+                if (!strieq(httpPath.str(), "/esp/login"))
```

Comment:
@wangkx do you require the path to be "/esp/login" for the login to work now?  I don't see where that happens.

Forcing the login to go to this URL to work will mean that the logging will always be hidden and will allow us to make other changes in the future.